.NET/Microsoft.Recognizers.Text/Parsers/IParser.cs

```diff
@@ -30,5 +30,9 @@ public ParseResult(ExtractResult er)
         // Output the value in string format.
         // It is used in some parsers.
         public string ResolutionStr { get; set; } = string.Empty;
+
+        public string Offset { get; set; } = null;
```

Comment:
If we change this here, are you sure they don't appear in other output?
This is specific to Ordinal numbers. Not sure if it should be in all ParseResults. Most entities won't have them.

Reply:
"current" and "end" is in the RelativeReferenceRelativeToMap.
Ordinal and Ordinal.relative have different implementation in this case.